### ISSUE TITLE
🐛 Bug: App Store popups not displaying logos

### ISSUE THREAD

**Issue by naamval** [nocodb/nocodb#6137]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

1. Go to App Store
2. Go to any app and click Install (or Edit)
3. The popup window does not display any image. It is trying to load (non-existent) /plugins/{app}.{extension} instead of /dashboard/plugins/{app}.{extension} (for instance, /plugins/minio.png instead of /dashboard/plugins/minio.png).

### Desired Behavior

The logo should be displayed

### Project Details

NocoDB running in Docker, arm64, tried with both mysql and sqlite3.

**Comment by wingkwong** [comment]
Able to reproduce.

![image](https://github.com/nocodb/nocodb/assets/35857179/9dd9ad79-5cc9-414c-a241-68a64f735c87)

**Comment by dstala** [comment]
@naamval can you confirm if you still see this issue in https://github.com/nocodb/nocodb/releases/tag/0.111.0?

**Comment by naamval** [comment]
> @naamval can you confirm if you still see this issue in https://github.com/nocodb/nocodb/releases/tag/0.111.0?

I just upgraded, still the same issue.

**Comment by Ponnada96** [comment]
Hello  @naamval and @dstala , I've just cloned the repo and got it running in my local environment. The app icons are now displaying in the pop-up.

**Comment by wingkwong** [comment]
@Ponnada96 The app icons can be shown locally but not in docker. Here's the screenshot from NocoDB hosting in Docker.

![image](https://github.com/nocodb/nocodb/assets/35857179/8cfd5725-02a6-44f3-8938-3c18dc4f911e)

**Comment by yblis** [comment]
yes, with local docker, I don't have any icons, but via my reverse proxy in https, everything works.


**With local docker in HTTP:**

![image](https://github.com/nocodb/nocodb/assets/10268910/83f4d7c5-c9e9-4c8f-bb5a-8782fb712d91)


**With reverse proxy Traefik & docker in HTTPS: **

![image](https://github.com/nocodb/nocodb/assets/10268910/cfead895-515d-4bb5-90da-ee9280dd6a25)

**Comment by o1lab** [comment]
@pranavxc

**Comment by dstala** [comment]
Related issue https://github.com/nocodb/nocodb/issues/6840

**Comment by o1lab** [comment]
Feel free to reopen if issue still persists.
